build.gradle

```diff
@@ -1,10 +1,10 @@
 buildscript {
     repositories {
-		mavenCentral()
```

Comment:
Well that leads to some problems, because newer versions of the build tools are only available through jcenter. Note that this change only affects dependencies not where your artifact is uploaded.